js/wee.routes.js

```diff
@@ -129,14 +129,10 @@
 
 					if (eq) {
 						// If ran is true then execute the route immediately
-						if (ran) {
-							if (object) {
-								_process(child, y, total, event, [opt, seg]);
```

Comment:
@leigeber where is this being accounted for now that it is removed? I assume that if we are dealing with an object, it would still need to be recursively processed, but I'm not seeing where this is happening exactly.